src/firmware/boards/robot_stm32h7/io/robot_status.c

```diff
@@ -0,0 +1,58 @@
+#include "firmware/boards/robot_stm32h7/io/robot_status.h"
+
+#include "firmware/boards/robot_stm32h7/io/proto_multicast.h"
+#include "firmware/boards/robot_stm32h7/io/proto_multicast_communication_profile.h"
+#include "shared/proto/robot_status_msg.nanopb.h"
+
+static TbotsProto_RobotStatus g_robot_status_msg;
```

Comment:
Can we initialize all the members at the declaration here? I think we can do nested named initialization.

Reply:
nanopb conveniently generates `TbotsProto_RobotStatus_init_zero`